app/src/main/java/com/shvants/shvantscalc/MainActivity.java

```diff
@@ -39,6 +41,9 @@
     public static final int UOE_TEXT_SIZE = 25;
     public static final int MEDIUM_LINE_LENGTH = 10;
     public static final int MAX_LINE_LENGTH = 15;
+    public static final int ONE = 1;
```

Comment:
You should extract this constants to another class.

Reply:
You mean all integer constants or ALL constants in class?